testing/homebank/APKBUILD

```diff
@@ -0,0 +1,37 @@
+# Contributor:
+# Maintainer: 7heo <7heo@mail.com>
+pkgname=homebank
+pkgver=5.1.5
+pkgrel=0
+sourcerev=56 # bump pkgrel when you change this. Get it form http://bazaar.launchpad.net/~mdoyen/homebank/5.1.x/changes
+pkgdesc="Free personal finance software, money management"
+url="http://homebank.free.fr/"
+arch="all"
+license="GPL"
+depends="libsoup"
+makedepends="bash gtk+3.0-dev libsoup-dev libofx-dev"
+install=""
+subpackages="$pkgname-lang"
+options="!check"
```

Comment:
This is an application, right? If upstream does not provide any tests, then you should add at least trivial check that just runs the executable to ensure that it’s not totally broken (e.g. `homebank --version`).